src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Context/FormattingContext.cs

```diff
@@ -293,15 +293,15 @@ public void AddInitialSuppressOperation(SuppressOperation operation)
                 valuePairs[i] = (operation, shouldSuppress: true, onSameLine);
             }
 
-            valuePairs.Do(v =>
+            foreach (var (operation, shouldSuppress, onSameLine) in valuePairs)
```

Comment:
📝 We could have made this change independently, but it's much more important now that `SuppressOperation` is larger.